readme.md

```diff
@@ -1,7 +1,9 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://unholysearcher
+.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://unholysearcher
+.github.io/layout_search-bar-airbnb/report/html_report/)
 
```

Comment:
For the future: here is a tip for a good project design in git. Next time, paste your link with all the formatting
![image](https://user-images.githubusercontent.com/70009205/233420273-cd308d91-624c-45bc-878e-b9cc1b244fa7.png)